README.md

```diff
@@ -238,7 +237,7 @@ Include kotlinx bintray repository for library:
 </repositories>
 ```
 
-You also can use JCenter.
+You can also use JCenter.
```

Comment:
Probably it's time to move JCenter to the first place, since even eap releases automatically synced.
Also, we support Maven Central now.